Ignore all ns

qpdpQ opened the pull request:
fix for ticket: https://github.ibm.com/IBMPrivateCloud/roadmap/issues/58091#issuecomment-55546849
we should ignore the role for nss operator itself it in all namespaces

Daniel-Fan reviewed (approved):
/lgtm

Daniel-Fan commented:
@qpdpQ 
Could we try `make code-dev` to fix the format issue?

Daniel-Fan commented:
Could we also update `  golangci-lint-version` to `1.51.x`? This is required for golang version `1.20` on prow cluster
https://github.com/IBM/ibm-namespace-scope-operator/blob/master/common/config/.golangci.yml#L3

qpdpQ commented:
lint error
```
level=warning msg="[runner] The linter 'deadcode' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused."
level=warning msg="[runner] The linter 'golint' is deprecated (since v1.41.0) due to: The repository of the linter has been archived by the owner. Replaced by revive."
level=warning msg="[runner] The linter 'varcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused."
level=warning msg="[runner] The linter 'interfacer' is deprecated (since v1.38.0) due to: The repository of the linter has been archived by the owner. "
level=warning msg="[runner] The linter 'structcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused."
```
remove them

Daniel-Fan commented on controllers/namespacescope_controller.go:
```diff
@@ -209,7 +210,6 @@ func (r *NamespaceScopeReconciler) UpdateConfigMap(instance *operatorv1.Namespac
 			if err := r.RestartPods(instance.Spec.RestartLabels, cm, instance.Namespace); err != nil {
 				return err
 			}
-			return nil
```
I think we could combine them together
```
return r.RestartPods(instance.Spec.RestartLabels, cm, instance.Namespace)
```
